x-pack/plugin/ccr/src/test/java/org/elasticsearch/xpack/ccr/ShardChangesIT.java

```diff
@@ -529,6 +543,53 @@ public void testAttemptToChangeCcrFollowingIndexSetting() throws Exception {
             "this setting is managed via a dedicated API"));
     }
 
+    public void testTransferMaxSeqNoOfUpdates() throws Exception {
```

Comment:
@dnhatn what's the added value of this test compared to asserting at all other tests that the MSU on the follower is the same as the primary after operations?

Reply:
Yep, I'll remove this test and reintroduce it when we have the optimization in the following engine.